routes/data/GET-getData.js

```diff
@@ -1,7 +1,17 @@
+const jwt = require('jsonwebtoken');
+
 const Data = require('../../models/data');
 
 module.exports = (app) => {
   app.get('/data/getData', (req, res) => {
+    const { token } = req.cookies || {};
+    try {
```

Comment:
Hi @matt8100,
Please remove the JWT verification code from getData, register and unregister routes and apply the middleware pattern as implemented by @Golem8 in PR #56. This will ensure that the code is maintainable. 

Thank you,
Harsimrat